src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate localDateTo = LocalDate.parse(dateTo, FORMATTER);
+        String infoSalary = null;
+        int[] salary = new int[names.length];
+
+        for (int i = 0; i < names.length; i++) {
+            for (int j = 0; j < data.length; j++) {
```

Comment:
```suggestion
        for (int i = 0; i < names.length; i++) {
        int salary = 0;
            for (int j = 0; j < data.length; j++) {
```
You can use `int salary` instead of `int[] salary` array
And better use for-each loops